### DISCUSSION TITLE
Handling 'large' sites (1000-2000 files)

### DISCUSSION THREAD

**Discussion by ghost** [retypeapp/retype#460] [Q&A]
How does Retype scale? What is build and 'watch' performance like when processing (say) up to 2000 files? Are there any figures?
TIA

**Comment by geoffreymcgill** [comment]
1000 to 2000 shouldn't be a problem. Retype will chew through that quickly. Once you get 1000+ pages, the client side performance will begin to degrade as the size of the left side-bar nav and search index grow large.
